systest/backup/common/utils.go

```diff
@@ -38,6 +49,12 @@ var (
 		"alpha2",
 		"alpha3",
 	}
+	namespaceId uint64
+)
+
+const (
+	accessJwtHeader = "X-Dgraph-AccessToken"
+	ShellToUse      = "bash"
```

Comment:
Does the scope of this constant need to be made public?

Reply:
**Resolved:** There is no need to make these constants public.